src/modules/github/components/connections/connections.tsx

```diff
@@ -44,14 +105,38 @@ export const Connections: FC = () => {
                   variant="ghost"
                   size="icon"
                   icon={<TrashIcon className="w-full h-full" />}
-                  onClick={() => {
-                    deleteInstallation(installation.id)
-                  }}
+                  onClick={() =>
+                    handleOpenDialog(installation.id, installation.owner)
+                  }
                 />
               </div>
             </li>
           ))}
       </ul>
+      <AlertDialog
+        open={dialogData.open}
+        onOpenChange={handleOpenChange}
+        onActionClick={() => handleActionClick()}
+        title={dialogData.title}
+        children={
+          <>
+            <div>
+              Are you sure you want to delete this{' '}
+              {dialogData.installationOwner} connection?
+              <br />
+              You can not undo this action?
+            </div>
```

Comment:
Use the translate function for this. And remove `?` at the end of "You can not undo this action"

Reply:
I resolved then how can I 'enter' between 'connection?' and 'You'? 🥲

Reply:
You can have two different translate

Reply:
What the hell! Oh my god.